tensorflow/core/util/mkl_heuristics.h

```diff
@@ -120,6 +120,71 @@ static double CalculateNodeMFlops(const AttrSlice& attrs,
   return -1;
 }
 
+static bool MatMulHeuristic(const Node* n) {
```

Comment:
Please document the return values of this function.

Reply:
Done